index.js

```diff
@@ -321,4 +372,56 @@ input DeleteOtherAnswerInput {
   parentAnswerId: ID!
 }
 
+input CreateRoutingRuleSetInput {
+  questionPageId: ID!
+}
+
+input UpdateRoutingRuleSetInput {
+  id: ID!
+  questionPageId: ID!
+}
+
+input DeleteRoutingRuleSetInput {
+  id: ID!
+}
+
+input CreateRoutingRuleInput {
+  routingRuleSet: ID!
+  operation: RoutingOperation
+  conditions: [ID]
+  goto: ID
```

Comment:
I wonder if `Goto` should be its own type. E.g. here isn't not obvious ID is a page ID. Also, gives us more flexibility to refactor/improve if everything is sort of namespaced in a type.